source/_integrations/binary_sensor.knx.markdown

```diff
@@ -22,11 +22,13 @@ Binary sensors are read-only. To write to the knx-bus configure an exposure [KNX
 
 The `knx` integration must be configured correctly, see [KNX Integration](/integrations/knx).
```

Comment:
I think we can delete this line now. `knx:` is mandatory now anyway.

Reply:
You still need to define the connection config correctly in order for it to work.

Reply:
in most cases automatic GatewayScanner should work fine 😉